app/src/main/java/com/geekskool/dhobi/Contracts/Contract.java

```diff
@@ -0,0 +1,99 @@
+package com.geekskool.dhobi.Contracts;
+
+import android.content.ContentUris;
+import android.net.Uri;
+import android.provider.BaseColumns;
+
+/**
+ * Created by manisharana on 17/2/17.
+ */
+public class Contract {
```

Comment:
DDL is happiest in a migration scheme of some sort. Migration tools will help you version your schema, migrate your data forward through different versions of your DB (and through time), and safely upgrade your software through various changes.